lib/goldencobra/version.rb

```diff
@@ -1,3 +1,3 @@
 module Goldencobra
-  VERSION = "2.3.11".freeze
+  VERSION = "2.3.12".freeze
```

Comment:
normalerweise besser versionsupdates erst im master vorzunehmen, da in einem branch ja eigentlich nicht klar ist, ob es vllt. andere features gibt, die vorher gemergt werden.